guides/v2.1/javascript-dev-guide/widgets/widget_accordion.md

```diff
@@ -179,6 +179,44 @@ $( "#element" ).accordion( "deactivate", 1 );
 $( "#element" ).accordion( "deactivate", [0,1]);
 ```
 
+## Events {#accordion_events}
+
+The accordion widget is subscribed to the same events as the [Magento Tabs widget] widget:
```

Comment:
Please, use the link markup for `[Magento Tabs widget]`. So we have [Tabs Widget](https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_tabs.html)